src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/EventHubAttribute.cs

```diff
@@ -27,5 +27,10 @@ public EventHubAttribute(string eventHubName)
         /// </summary>
         [AutoResolve]
         public string EventHubName { get; private set; }
+
+        /// <summary>
+        /// Optional connection name. If missing, tries to use a registered event hub sender.
+        /// </summary>
+        public string Connection { get; set; }
```

Comment:
Yeah, I considered it but had some misconceptions about how AppSettingAttribute handled null values. Investigated further, it should work, I'll make the change.